docs/source/code-generation/codegen-configuration.mdx

```diff
@@ -447,26 +443,26 @@ The top-level properties are:
 	"pruneGeneratedFiles": true,
 	"markOperationDefinitionsAsFinal": true,
 	"schemaCustomization" : {
-    "customTypeNames" : {
-      "MyEnum" : {
-        "enum" : {
-          "cases" : {
-            "MyCase" : "CustomCase"
-          },
-          "name" : "CustomEnum"
-        }
-      },
-      "MyObject" : "CustomAnimal",
-      "MyInputObject" : {
-        "inputObject" : {
-          "fields" : {
-            "myField" : "customField"
-          },
-          "name" : "CustomInputObject"
-        }
-      }
-    }
-  }
+		"customTypeNames" : {
```

Comment:
The diff here make it look like nothing has changed but the alignment in the JSON sample in the docs is not correct.